set.mm

```diff
@@ -440817,65 +440817,65 @@ orthogonal vectors (i.e. whose inner product is 0) is the sum of the
   althtmldef ".ef" as ".ef";
   latexdef ".ef" as "{\rm ef}";
 htmldef "Vtx" as 'Vtx';
- althtmldef "Vtx" as 'Vtx';
- latexdef "Vtx" as "{\rm Vtx}";
+  althtmldef "Vtx" as 'Vtx';
+  latexdef "Vtx" as "{\rm Vtx}";
 htmldef "iEdg" as 'iEdg';
- althtmldef "iEdg" as 'iEdg';
- latexdef "iEdg" as "{\rm iEdg}";
-htmldef "UHGraph" as ' UHGraph ';
- althtmldef "UHGraph" as ' UHGraph ';
- latexdef "UHGraph" as "{\rm UHGraph}";
-htmldef "USHGraph" as ' USHGraph ';
- althtmldef "USHGraph" as ' USHGraph ';
- latexdef "USHGraph" as "{\rm USHGraph}";
-htmldef "UPGraph" as ' UPGraph ';
- althtmldef "UPGraph" as ' UPGraph ';
- latexdef "UPGraph" as "{\rm UPGraph}";
-htmldef "UMGraph" as ' UMGraph ';
- althtmldef "UMGraph" as ' UMGraph ';
- latexdef "UMGraph" as "{\rm UMGraph}";
+  althtmldef "iEdg" as 'iEdg';
+  latexdef "iEdg" as "{\rm iEdg}";
+htmldef "UHGraph" as 'UHGraph';
+  althtmldef "UHGraph" as 'UHGraph';
+  latexdef "UHGraph" as "{\rm UHGraph}";
+htmldef "USHGraph" as 'USHGraph';
+  althtmldef "USHGraph" as 'USHGraph';
+  latexdef "USHGraph" as "{\rm USHGraph}";
+htmldef "UPGraph" as 'UPGraph';
+  althtmldef "UPGraph" as 'UPGraph';
+  latexdef "UPGraph" as "{\rm UPGraph}";
+htmldef "UMGraph" as 'UMGraph';
+  althtmldef "UMGraph" as 'UMGraph';
+  latexdef "UMGraph" as "{\rm UMGraph}";
 htmldef "Edg" as 'Edg';
- althtmldef "Edg" as 'Edg';
- latexdef "Edg" as "{\rm Edg}";
-htmldef "USPGraph" as ' USPGraph ';
- althtmldef "USPGraph" as ' USPGraph ';
- latexdef "USPGraph" as "{\rm USPGraph}";
-htmldef "USGraph" as ' USGraph ';
- althtmldef "USGraph" as ' USGraph ';
- latexdef "USGraph" as "{\rm USGraph}";
-htmldef "SubGraph" as ' SubGraph ';
- althtmldef "SubGraph" as ' SubGraph ';
- latexdef "SubGraph" as "{\rm SubGraph}";
-htmldef "FinUSGraph" as ' FinUSGraph ';
- althtmldef "FinUSGraph" as ' FinUSGraph ';
- latexdef "FinUSGraph" as "{\rm FinUSGraph}";
-htmldef "NeighbVtx" as ' NeighbVtx ';
- althtmldef "NeighbVtx" as ' NeighbVtx ';
- latexdef "NeighbVtx" as "{\rm NeighbVtx}";
+  althtmldef "Edg" as 'Edg';
```

Comment:
I agree with removing the blanks for most of the "*Graph" classes, because these are not used with arguments/operands.

For SubGraph and NeighbVtx, however, I disagree: they are almost always used as relations ` S SubGraph G ` resp. operations ` ( G NeighbVtx N ) `. According to the conventions (item "Typography conventions"), the representations should  include leading and trailing blanks. Maybe this convention has to be enhanced for relations (see, for example, the html representation for ~QG.